src/planner/PlanNode.h

```diff
@@ -232,6 +232,7 @@ class PlanNode {
 
     void clone(const PlanNode &node) {
         // TODO maybe shall copy cost_ and dependencies_ too
+        id_ = node.id_;
```

Comment:
how to differentiate these two plan nodes by id if they have same node ids?

Reply:
My fault...Has fixed